man/subscription-manager.8

```diff
@@ -83,58 +83,55 @@ must be passed as system arguments in a non-interactive session.
 4. auto-attach
 
 .IP
-5. remove
```

Comment:
Same comment here. Please, jut remove `remove` command from the list and do not change numbers of other commands. We will fix this later.